core/modules/catalog-next/helpers/categoryHelpers.ts

```diff
@@ -14,3 +14,14 @@ export const _prepareCategoryPathIds = (category: Category): string[] => {
   if (!category || !category.path) return []
   return category.path.split('/')
 }
+
+export const getSearchOptionsFromRouteParams = (params: { [key: string]: string } = {}): Object => {
+  const filterableKeys = ['url-key', 'slug', 'id']
```

Comment:
I'd move this one to config - to have the filterable keys editable in the config

Reply:
Yes indeed, that'd be more clean.
I've added them to the configs.